### ISSUE TITLE
As a user, I want reminders for team review requests to @mention a Slack group

### ISSUE THREAD

**Issue by abinoda** [pullreminders/backlog#148]
Requested by RothAndrew, mwallace72, etc

**Comment by israel-tk** [comment]
It would be cool if we could set team review requests reminders to target a certain channel.

For example, all review requested for team "flights" have reminders go to channel #flights_squad in Slack.

**Comment by abinoda** [comment]
@israel-tk This should already be possible. Under your team reminder settings, do you see the "Filter by GitHub Team" option? That lets you filter reminders to a particular channel so it only reminds the team or team members based on your selection.

Let me know if you have questions or trouble setting it up.

**Comment by israel-tk** [comment]
@abinoda thanks for the info, didn't see that option! I understand this will show up reminders for open PRs whenever at least one of the requested reviewers is part of the specified team, right? Not only when who was selected for review was the team itself. Am I mistaken?

**Comment by abinoda** [comment]
@israel-tk Correct, it works for both cases (team or team member being requested).

**Comment by jpbufe3** [comment]
~@abinoda - If I'm understanding correctly, the only way to notify a team on Slack is by using the "team reminders" feature and filtering to a GitHub team. Would it be possible to add functionality that supports real-time notifications for any PRs assigned to that GitHub team that notifies a Slack user group? Essentially I'd like to have "DM settings" for a GitHub team that notifies a Slack user group. Thanks!~

Update - I just realized that you can specify your DM settings to include teams you're on, which should be sufficient for my use case. I could imagine there still being a benefit to linking a GitHub team with a Slack user group in some work flow, but might not be necessary. Sorry for the noise!
